pagerduty/provider.go

```diff
@@ -99,7 +105,17 @@ func handleNotFoundError(err error, d *schema.ResourceData) error {
 }
 
 func providerConfigure(data *schema.ResourceData, terraformVersion string) (interface{}, error) {
+	var ServiceRegion = data.Get("service_region").(string)
```

Comment:
Let's just always set the value of `ServiceRegion` to lowercase.

Reply:
Done